src/applications/personalization/profile/tests/components/notification-settings/NotificationSettings.unit.spec.jsx

```diff
@@ -133,11 +133,9 @@ describe('<NotificationSettings />', () => {
       path: '/profile/notifications',
     });
 
-    expect(await view.findByText('Add a mobile phone number to your profile'))
-      .to.exist;
+    expect(await view.findByTestId('add-mobile-phone-link')).to.exist;
 
-    expect(view.queryByText('Add an email address to your profile')).to.not
-      .exist;
+    expect(view.queryByTestId('add-email-address-link')).to.not.exist;
```

Comment:
Good question! I like to use the table here to help me https://testing-library.com/docs/react-testing-library/cheatsheet/

`queryBy` doesn't throw an error when it doesn't find something vs all the rest throw errors, so that has been my go to for when I am checking that something does not exist in the UI.